.travis.yml

```diff
@@ -31,7 +31,7 @@ script:
 
 env:
   matrix:
-    - PYTHONASYNCIODEBUG=1 AIOHTTP_NO_EXTENSIONS=0
-    - PYTHONASYNCIODEBUG=0 AIOHTTP_NO_EXTENSIONS=0
-    - PYTHONASYNCIODEBUG=1 AIOHTTP_NO_EXTENSIONS=1
-    - PYTHONASYNCIODEBUG=0 AIOHTTP_NO_EXTENSIONS=1
+    - PYTHONASYNCIODEBUG=nonemptystring AIOHTTP_NO_EXTENSIONS=
```

Comment:
I don't think this affects only `aiohttp` contributors. Since this is Open Source project it's common to use *any* part of it in other projects. For example I used this `.travis.yml` as reference in `aiohttp_cors`, and most probably will use again in other `asyncio`-based project.

What about single character + comment?